scripts/airdrop.js

```diff
@@ -0,0 +1,30 @@
+const assert = require('assert')
+
+const parseCsv = require('../test/utils').parseCsv
+const Token = artifacts.require("./contracts/MaticToken.sol")
+const Airdrop = artifacts.require("./contracts/MaticTokenAirdrop.sol")
+const SCALING_FACTOR = web3.utils.toBN(10 ** 18)
+
+async function fireTx() {
+  let { recipients, amounts, airdropSupply } = await parseCsv(web3)
```

Comment:
we might wanna add one check here for `total amounts`===`total amount from csv` in case of any error in formatting.
what do you think @atvanguard? maybe take input from cli?

Reply:
Ok, I know that the total is 48198. I'll hardcode that assertion here. Also we'll transfer exactly 48198 tokens to this faucet.